dev/com.ibm.ws.kernel.service/src/com/ibm/wsspi/kernel/service/utils/SerializableSchedule.java

```diff
@@ -0,0 +1,258 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.wsspi.kernel.service.utils;
+
+import java.io.Serializable;
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.time.LocalTime;
```

Comment:
I'm sorry that I didn't think of this during the design review, but I'm concerned about the use of local time because it requires the user to have knowledge of which time zone(s) their servers are running in and forces them to think in that time zone versus the time zone that their business is running in, which will be more natural for them. I wonder if we should add a (optional?) time zone/zone id to the configuration, `MON-FRI 9:00-17:00 America/Chicago` and operate in ZonedDateTime.  Time zones like CDT or CST are more familiar to most people, but would lead to confusion when crossing over the daylight saving time boundary outside of what they pertain to.